--- nosegae.py	2011-11-09 07:15:30.000000000 +0100
+++ nosegae_new.py	2014-04-06 14:49:17.000000000 +0200
@@ -81,7 +81,7 @@
             fix_sys_path() # wipes out sys.path
             sys.path.extend(saved_path) # put back our previous path
             
-            from google.appengine.tools import dev_appserver
+            from google.appengine.tools import old_dev_appserver as dev_appserver
             from google.appengine.tools.dev_appserver_main import \
                 DEFAULT_ARGS, ARG_CLEAR_DATASTORE, ARG_LOG_LEVEL, \
                 ARG_DATASTORE_PATH, ARG_HISTORY_PATH
